spec/board_spec.rb

```diff
@@ -0,0 +1,84 @@
+require_relative '../lib/board'
+require_relative '../lib/token'
+
+describe 'Board' do
+    include Token
```

Comment:
Why do you need to `include Token` in these tests?

Reply:
I included Token so I dont have hard coded symbols/strings throughout my code. It'll always match the tokens that I use in the production code.
is that ok? or are you saying that the use of Tokens are ok , but not needed in this specific set of tests.

Reply:
This is my guess, but I'd say the concept of `include`ing or `extend`ing can raise some flags for folks because they change the method / instance surface area of the object you're working on.  So, basically, if the other module changes at some point, you have to make sure there's no collisions or unpredictable dependencies.  Good test discipline makes that not as much of a problem, but it can still come up and bite you at random times.

The `extend self` pattern I describe above helps with that because it calls out where every value is coming from, which at least lowers the barrier for a future reader who might not know where everything is mapped to/from.